roop/core.py

```diff
@@ -94,6 +97,17 @@ def pre_check():
     if '--all-faces' in sys.argv or '-a' in sys.argv:
         roop.globals.all_faces = True
 
+def pre_check_codeformer():
+    if '--upscale' in sys.argv or args['upscale']:
+        codeformer_model_path = os.path.join('codeformer', 'weights', 'codeformer', 'codeformer.pth')
+        facelib_model_path = os.path.join('codeformer', 'weights', 'facelib', 'detection_Resnet50_Final.pth')
+        facelib_model_path2 = os.path.join('codeformer', 'weights', 'facelib', 'parsing_parsenet.pth')
+        realesrgan_model_path = os.path.join('codeformer', 'weights', 'realesrgan', 'RealESRGAN_x2plus.pth')
+        if os.path.isfile(codeformer_model_path) and os.path.isfile(facelib_model_path) and os.path.isfile(facelib_model_path2) and os.path.isfile(realesrgan_model_path):
+            print("Passed model check")
```

Comment:
Output only if on failure...

Reply:
I checked the source code of codeformer-pip，in fact the package will auto download the model files from github if the models dont exist.

is checking again necessary?

I thought its just useful for people without internet connection
[Screenshot_20230603_182921_com microsoft emmx](https://github.com/s0md3v/roop/assets/28835093/684b1071-3cbc-486c-811b-3948dafcaeb4)
("Passed model check")
```

Reply:
Then we can remove that check again - I did not know that... sorry

Reply:
Can you reach out to me via Discord?

Reply:
OK，after I back home